index.html

```diff
@@ -79,7 +79,7 @@ <h5 class="quote-author" id="quoteAuthorTag">- Steve Jobs</h5>
     <script src="./assets/js/main.js"></script>
     <script>
       var color_selection=document.querySelectorAll('.select_color')
-      var color_array=['#81CACF','#E68398','#7EDE80','rgb(229 229 62)'];
+      var color_array=['rgb(25 116 122 / 30%)','rgb(154 72 89/ 30%)','rgb(95 190 97 / 30%)','rgb(255 255 0 / 30%)'];
```

Comment:
Can you make the color array element more brighter? i mean more than 30%?

Reply:
yes i was thinking the same do u mind suggesting me or should i choose on my own?

Reply:
You can use 50% or 60% or
you can choose your own 😄

Reply:
done please check and possible merge.

Reply:
got it